clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -65,6 +65,9 @@
     /* The v2 metadata request is the same as v1. An additional field for cluster id has been added to the v2 metadata response */
     public static final Schema METADATA_REQUEST_V2 = METADATA_REQUEST_V1;
 
+    /* The v3 metadata request is the same as v1 and v2. An additional field for throttle time has been added to the v3 metadata response */
```

Comment:
I have added unit test in `ApiKeysTest` that checks that all responses except those explicitly excluded contain a field named `throttle_time_ms`. There is a comment in the test as well to ensure that new requests either contain the field or are manually added to the test's exclusion list.